demo-shell-ng2/app/components/form/form-demo.component.css

```diff
@@ -0,0 +1,3 @@
+.form-container {
```

Comment:
this is an end application (demo shell), it does not matter whether it has prefix or not :)